### ISSUE TITLE
SVG engine on steroids

### ISSUE THREAD

**Issue by MarcoAbi** [amcharts/amcharts4#711]
Hi all.
First of all, my compliments for this wonderful piece of code that amcharts is made of. Really a great job. 
And the SVG engine is just great! 

Now, what I’m just thinking is that would be great if the SVG engine you just created would have the ability to allow user to interact more with elements. 

Looking the demos I’ve seen already nice stuff but I think the SVG engine can have some other little extras. What I have in mind is fabric.js interaction capabilities (such as focusing on click, and a guide around the object in order to resize/move/rotate the element). 
This feature may be the ground for so many new use cases that may be really interesting such as change chart look and feel or behavior just clicking, moving it around in the container along with other charts. 

What do you think about?

I’ll be more than happy to contribute on this enhancement. Anyway, i’d be thankful if you can point me out where I can start digging to add such features. 

Basically I would like to start to show a boundary around the object with a little frame around the object, with little squares on the respective corner to allow resizing and a rotating pointer on top. 

Cheers!!
Marco
